advanced_alchemy/operations.py

```diff
@@ -87,19 +87,3 @@ def visit_merge(element: Merge, compiler: StrSQLCompiler, **kw: Any) -> str:
         sql_text += f" {clauses}"
 
     return sql_text
-
-
-class InsertFromSelect(Executable, ClauseElement):
-    inherit_cache = False
-
-    def __init__(self, table: FromClause, select: Select) -> None:
-        self.table = table
-        self.select = select
-
-
-@compiles(InsertFromSelect)  # type: ignore[no-untyped-call, misc]
-def visit_insert_from_select(element: InsertFromSelect, compiler: StrSQLCompiler, **kw: Any) -> str:
-    return "INSERT INTO {} ({})".format(
-        compiler.process(element.table, asfrom=True, **kw),
-        compiler.process(element.select, **kw),
-    )
```

Comment:
This is unrelated but unused.  Removing.  This example is also in the official SQLA docs.